plugins/com.google.cloud.tools.eclipse.test.logback/logback.xml

```diff
@@ -0,0 +1,12 @@
+<configuration scan="true"> 
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender"> 
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
```

Comment:
I don't know much about this settings file, but I wonder if we can leave this out and use the default pattern (for the sake of not adding something we don't really need).